site/documentation/content/deployment/helm-based-deployment.md

```diff
@@ -106,58 +106,41 @@ prometheus-operated                     ClusterIP      None             <none>
 ~~~
 
 The listing above has been retrieved from a Minikube cluster that emulates a load balancer via the `minikube tunnel` command (refer to the [Minikube Networking docs](https://github.com/kubernetes/minikube/blob/master/docs/networking.md#loadbalancer-emulation-minikube-tunnel) for details).
-The service endpoints are accessible on the *EXTERNAL-IP* addresses and the corresponding *PORT(S)*, e.g. 8080 for the HTTP adapter and 28080 for the device registry.
+The service endpoints can be accessed at the *EXTERNAL-IP* addresses and corresponding *PORT(S)*, e.g. 8080 for the HTTP adapter (*hono-adapter-http-vertx*) and 28080 for the device registry (*hono-service-device-registry*).
 
-### Starting a Consumer
-
-Data published by devices is usually consumed by downstream applications which connect directly to the AMQP messaging network (represented by the `hono-dispatch-router-ext` service.
-The following command starts a consumer from the command line:
+The following command assigns the IP addresses of the exposed services to environment variables so that they can easily be used from the command line:
 
 ~~~sh
 # in directory: eclipse-hono-$VERSION/deploy/
 eval $(services.sh)
-
-# in the directory where the Hono command line client has been downloaded to:
-java -jar hono-cli-*-exec.jar --hono.client.host=$AMQP_NETWORK_IP --hono.client.port=15672 --hono.client.username=consumer@HONO --hono.client.password=verysecret --spring.profiles.active=receiver
 ~~~
 
-This will start a new consumer that logs all telemetry and event messages to the console.
-
-### Uploading Telemetry
-
-In another console, run the following command in order to put the IP addresses of Hono's service endpoints to
-some environment variables:
+The following command can then be used to check for the existence of the *DEFAULT_TENANT* which is created as part of the installation:
 
 ~~~sh
-# in directory: eclipse-hono-$VERSION/deploy/
-eval $(services.sh)
-~~~
-
-In order to upload telemetry data to Hono, a device needs to be registered with the system first.
-The following command registers a device with ID `4711` in the example device registry that is deployed with Hono:
+curl -sIXGET http://$REGISTRY_IP:28080/tenant/DEFAULT_TENANT
```

Comment:
Although the command seems to work, I guess there should be a space after the `X`:
`curl -sIX GET`